### DISCUSSION TITLE
Searching user whether the specified user attributes exists or not

### DISCUSSION THREAD

**Discussion by Aleson95** [keycloak/keycloak#13716] [Q&A] [answered]
Is there anyway to search user whether the specified user attributes exists or not, I don't care about the value I just wanted to search all users that has this user attributes.

**Comment by hmlnarik** [comment, marked as answer]
No. If you would like to have this capability, feel free to raise a RFE for the new store. Note please that this is unlikely to be supported in the legacy store, and implementation in the new store would need to wait for generic search to be extracted to `UserProvider`.
